exercises/spiral-matrix/spiral_matrix_test.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+
+from spiral_matrix import spiral
+
+# Tests are based on the version 1.0.0 of:
+# https://github.com/exercism/problem-specifications/blob/master/exercises/spiral-matrix/canonical-data.json
```

Comment:
This comment should be ```# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0``` as per #784. Could you also move it down by a line? (ie. two blank lines before, one after)

Reply:
You want me to put something between those two slashes, or not?

Reply:
@Average-user, no, it's just a shorthand way of saying that we are skipping some uninteresting directories in the middle. The full path would be `problem-specifications/exercises/spiral-matrix/canonical-data.json`, which is a bit long to include in a comment, and the middle directories are fairly obvious.